subscriber/synapse-subscriber-kafka/src/main/java/io/americanexpress/synapse/subscriber/kafka/annotation/KafkaSubscriber.java

```diff
@@ -0,0 +1,28 @@
+package io.americanexpress.synapse.subscriber.kafka.annotation;
+
+import org.springframework.core.annotation.AliasFor;
+import org.springframework.stereotype.Component;
+
+import java.lang.annotation.*;
```

Comment:
can we import the individual annotations instead of *?

Reply:
done